src/System.Dynamic.Runtime/tests/Dynamic.Simple/DelegateTest.cs

```diff
@@ -24,5 +24,38 @@ public static void DelegateTest_RunTest()
             int result = d(4, 6);
             Assert.Equal(24, result);
         }
+
+        private static int Foo(string s, dynamic d, object o)
+        {
+            return s?.Length + d?.ToString()?.Length + o?.ToString()?.Length;
+        }
```

Comment:
It is as of #6181,and indeed there's `nameof` all over the place now. This test is my little act of celebration.